fuzz/fsfuzzer.py

```diff
@@ -0,0 +1,88 @@
+#!/usr/bin/env python
+
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation; either version 2 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+#
+# See LICENSE for more details.
+#
+# Copyright: 2016 IBM
+# Author: Praveen K Pandey <praveen@linux.vnet.ibm.com>
+#
+# Based on code by Martin Bligh <mbligh@google.com>
+#   copyright: 2006 Google
+#   https://github.com/autotest/autotest-client-tests/tree/master/fsfuzzer
+
+import os
+
+from avocado import Test
+from avocado import main
+from avocado.utils import archive
+from avocado.utils import process
+from avocado.utils import build
+from avocado.utils.software_manager import SoftwareManager
+
+
+class Fsfuzzer(Test):
+
+    """
+    fsfuzzer is a file system fuzzer tool. This test simply runs fsfuzzer
+    Fuzzing is slang for fault injection via random inputs. The goal is to
+    find bugs in software without reading code or designing detailed test
+    cases.
+       fsfuzz will inject random errors into the files systems
+       mounted. Evidently it has found many errors in many systems.
+    """
+
+    def setUp(self):
+        '''
+        Build fsfuzzer
+        Source:
+        http://people.redhat.com/sgrubb/files/fsfuzzer-0.7.tar.gz
```

Comment:
Hey, I talked to Steve and the official `fsfuzzer` is now located at https://github.com/stevegrubb/fsfuzzer and he applied the import patch. So perhaps you could send all the patches as PR to that repo so eventually we don't need to use any workarounds at all, only the latest master:

https://github.com/stevegrubb/fsfuzzer/archive/master.zip